|What is the title of your workflow?|What type of workflow is it?|If "Other", please specify the workflow type here|Short description|Please provide your name|What is your group+unit or service?|What is your EMBL site?|How many people maintain the workflow?|Which workflow engine is used to execute your workflow?|Is your workflow experimental or production grade?|How do you version your workflow? |How often do you release new updates?|How are you testing your pipeline?|Are some CI/CD (automated) tests applied to the workflow?|What do you use as CI/CD system?|If "Other" type of CI/CD, please specify the type here|Do you test the workflow to be compliant with new engine releases?|How do you test the workflow before releasing updates?|What technology are you using to manage dependencies of the workflow? (if multiple apply, choose the first and add a comment about other solutions in the last column)|If using containers, how do you create and version your containers?|Do you provide one container per rule or a workflow container?|Does you workflow require specialized hardware?|If yes, can you specify?|Is your workfklow depending on any external tools,  services, API or databases ? (Ensembl, NCBI, ...)|If so, which ones?|Does your workflow require manual steps prior to execution (choose the first option that applies)?|What output(s) is/are generated in the final step of your workflow? (choose the first option that applies)|How many execution paths does your workflow allow?|How do you handle errors or unexpected behaviors in the workflow?|How often do you execute this workflow?|Given the average input, how much time does the workflow need to execute?|How many variables or pre-configuration settings are needed to execute your workflow?|Does the workflow have any data requirements or pre-requisites (sequencing barcodes, indices, databases,...)?|If yes, do you provide a script or a guideline that help fullfiling these requirements?|Is the workflow publicly available?|Is the workflow referenced on a  (public/private) registry?|If yes, can you provide a link?|How many groups use your workflow besides your own?|Is your workflow known/used by other researchers at EMBL?|Who is the primary user of the workflow?|Do you document and provide examples on how to use your workflow?|Is a diagram of the steps in your workflow part of the documentation?|How do you provide support to users of your workflow? (choose the most often used if multiple apply)|How is your workflow licensed?|Do you use any documentation engine (sphinx, doxygen,...)?|Do you provide test input data?|What standard data formats does your workflow accept and output? (separate multiple formats by commas)|What is the primary output data type or format?|Are there any specific input/output file naming conventions used in the workflow?|Does the workflow connect to any data management system (e.g. STOCKS) to retrieve or store data and metadata?|What programming language(s) are used in the workflow to do custom steps (outside bioinfo classic tools)?|
|:----|:----|:----|:----|:----|:----|:----|:----|:----|:----|:----|:----|:----|:----|:----|:----|:----|:----|:----|:----|:----|:----|:----|:----|:----|:----|:----|:----|:----|:----|:----|:----|:----|:----|:----|:----|:----|:----|:----|:----|:----|:----|:----|:----|:----|:----|:----|:----|:----|:----|:----|
|mosaicatcher-pipeline|Genomics and comparative genomics| |Structural variant calling pipeline from single-cell Strand-seq data|Thomas Weber|Korbel Group - Genome Biology|EMBL Heidelberg|1|Snakemake|Production|Semantic Versioning|Monthly|GitHub actions| | | | | |Container (Docker, Singularity)| | |No| | | |None, the workflow handles all the above|Final results or report (plots / tables)|2 or more (e.g. alternative tools for key steps)| |Weekly|1-6 hours| | | | |Snakemake workflow catalog| |6-20|Yes|My group|Yes|Yes|Email|BSD/MIT| |Yes|FastQ, BAM| | |No| |
|ashleys-qc-pipeline|Genomics and comparative genomics| |Strand-Seq data preprocessing|Thomas Weber|Korbel Group - Genome Biology|EMBL Heidelberg|1|Snakemake|Production|Semantic Versioning|Monthly|GitHub actions| | | | | |Container (Docker, Singularity)| | |No| | | |None, the workflow handles all the above|Modified files to be used in other analysis|2 or more (e.g. alternative tools for key steps)| |Weekly|1-6 hours| | | | |Snakemake workflow catalog| |6-20|Yes|My group|Yes|Yes|Email|BSD/MIT| |Yes|FastQ, BAM| | |No| |
|strandseq-pooling-sample-id|Genomics and comparative genomics| |Strand-Seq pooling library identification from SNP genotyping |Thomas Weber|Korbel Group - Genome Biology|EMBL Heidelberg|1|Snakemake|Experimental|I do not version my workflow|Other|I'm not testing| | | | | |Virtual environments (Conda, Venv, Renv,...)| | |No| | | |Downloading large files or databases|Modified files to be used in other analysis|Single (linear)| |Other|1-6 hours| | | | |No, is not available| |1-5|No|My group|No|No|Email|Not open-sourced/open-licensed| |No|BAM, VCF| | |No| |
|European Nucleotide Archive Pathogen Analysis System|Genomics and comparative genomics| |Production system for pathogen analysis (e.g. SARS-CoV-2 and Mpox)|David Yuan|European Nucleotide Archive|EMBL-EBI|4|Nextflow|Production|Semantic Versioning|Daily|Manual testing (occasionally)| | | | | |Container (Docker, Singularity)| | |No| | | |None, the workflow handles all the above|Modified files to be used in other analysis|2 or more (e.g. alternative tools for key steps)| |Daily|<1 hour| | | | |Is public/privately available but not through a registry| |1-5|No|My group|Yes|Yes|Email|Apache| |Yes|FastQ, BAM, VCF| | |Yes| |
|ChEBI Ontology Generator|Metabolomics and chemical biology| |It creates an ontology (the ChEBI ontology) and its variants from a PostgreSQL database|Carlos Moreno|Chemical Biology Serivices|EMBL-EBI|1|Other|Production|Semantic Versioning|Monthly|Gitlab CI/CD| | | | | |Container (Docker, Singularity)| | |No| | | |None, the workflow handles all the above|Final results or report (plots / tables)|Single (linear)| |Daily|<1 hour| | | | |DockStore| |1-5|No|My group|Yes|No|Email|Other| |No|OWL, OBO, GZ| | |No| |
|nuclear_phentotyping|Bioimaging| |Turns images of nuclei into classical feature vectors ina dataframe. Also trains Deep learning models for the cell segmentation|Craig Russell|Uhlmann|EMBL-EBI|1|Snakemake|Experimental|Semantic Versioning|Monthly|GitHub actions| | | | | |Container (Docker, Singularity)| | |Graphical Processing Unit (GPU)| | | |None, the workflow handles all the above|Modified files to be used in other analysis|2 or more (e.g. alternative tools for key steps)| |Monthly|1-6 hours| | | | |No, is not available| | |No|Myself|No|No|Bug tracker|Other| |Yes| | | |No| |
|Analysis workflow for new experiments processing in Expression Atlas|Transcriptomics| |Snakemake workflow for processing, reprocessing and recalculations steps for RNA-seq, microarrays and proteomics experiments in Expression Atlas|Pedro Madrigal|Papatheodorou (Gene Expression Team)|EMBL-EBI|3|Snakemake|Production|Semantic Versioning|Weekly|Jenkins| | | | | |Virtual environments (Conda, Venv, Renv,...)| | |No| | | |Parameterization / Settings|Final results or report (plots / tables)|2 or more (e.g. alternative tools for key steps)| |Weekly|1-6 hours| | | | |Is public/privately available but not through a registry| |1-5|No|My group|Yes|No|Bug tracker|Not open-sourced/open-licensed| |No|FastQ, Curated metadata| | |Yes| |
|Single-cell Expression Atlas control workflow for single-cell expression data analysis|Other|Single Cell Transcriptomics|Master Nextflow workflow for running each component of the SMART-seq and droplet pipelines at Single Cell Expression Atlas |Pedro Madrigal|Papatheodorou (Gene Expression Team)|EMBL-EBI|3|Nextflow|Production|Semantic Versioning|Weekly|Jenkins| | | | | |Virtual environments (Conda, Venv, Renv,...)| | |No| | | |Parameterization / Settings|Final results or report (plots / tables)|2 or more (e.g. alternative tools for key steps)| |Weekly|6-24 hours| | | | |Is public/privately available but not through a registry| |1-5|No|My group|Yes|Yes|Bug tracker|Not open-sourced/open-licensed| |No|FastQ, Curated metadata| | |Yes| |
|2P-ca-prossecing pipeline|Neurobiology and neuroinformatics| |Load raw image stacks, pass them through both suite2p and caiman processing|Tristan Wiessalla|Asari|EMBL Rome|1|Snakemake|Experimental|I do not version my workflow|Other|Gitlab CI/CD| | | | | |Virtual environments (Conda, Venv, Renv,...)| | |No| | | |Downloading large files or databases|Final results or report (plots / tables)|2 or more (e.g. alternative tools for key steps)| |Monthly|24-72 hours| | | | |No, is not available| | |No|Myself|No|No|I don't|BSD/MIT| |No|16bit big endian raw stack| | |No| |
|Custom MetaboAnalystR implementation|Metabolomics and chemical biology| |Analysis of untargeted metabolomics starting from peak intensities ("counts") to statistical analyses and dimensionality reduction approaches|Bobby Ranjan|Hackett|EMBL Rome|1|Snakemake|Experimental|I do not version my workflow|I'm not releasing updates|I'm not testing| | | | | |Virtual environments (Conda, Venv, Renv,...)| | |No| | | |None, the workflow handles all the above|Final results or report (plots / tables)|2 or more (e.g. alternative tools for key steps)| |Monthly|6-24 hours| | | | |No, is not available| |1-5|Yes|Myself|No|No|I don't|Not open-sourced/open-licensed| |Yes|mzML, TXT| | |No| |
|Bulk RNA-seq|Transcriptomics| |QC, alignment, quantification and differential expression for protein coding genes. TE-specific alignments and quantification|Francesco Tabaro|Boulard|EMBL Rome|1|Snakemake|Experimental|I do not version my workflow|I'm not releasing updates|Manual testing (occasionally)| | | | | |Container (Docker, Singularity)| | |No| | | |Downloading large files or databases|Final results or report (plots / tables)|2 or more (e.g. alternative tools for key steps)| |Other|6-24 hours| | | | |No, is not available| |1-5|Yes|Myself|No|Yes|I don't|Not open-sourced/open-licensed| |Yes|Fastq| | |No| |
|Chip-seq|Genomics and comparative genomics| |QC, alignment, peak calling, peak annotation, differential binding|Francesco Tabaro|Boulard|EMBL Rome|1|Snakemake|Experimental|I do not version my workflow|I'm not releasing updates|Manual testing (occasionally)| | | | | |Other| | |No| | | |Downloading large files or databases|Final results or report (plots / tables)|Single (linear)| |Other|24-72 hours| | | | |No, is not available| |1-5|Yes|Myself|No|No|I don't|Not open-sourced/open-licensed| |No|Fastq| | |No| |
|MethylSnake|Genomics and comparative genomics| |QC, alignment, 5MC detection, differential methylation analysis|Francesco Tabaro|Boulard|EMBL Rome|1|Snakemake|Experimental|Release date|Other|Manual testing (occasionally)| | | | | |Container (Docker, Singularity)| | |No| | | |Downloading large files or databases|Final results or report (plots / tables)|Single (linear)| |Other|6-24 hours| | | | |Snakemake workflow catalog| |1-5|No|Myself|Yes|Yes|Bug tracker|GNU GPL| |No|Fastq| | |No| |
|Inspection|Genomics and comparative genomics| |Identification of antibiotic resistance peptides|Francesco Tabaro|Boulard|EMBL Rome|1|Snakemake|Experimental|I do not version my workflow|Other|Manual testing (occasionally)| | | | | |Virtual environments (Conda, Venv, Renv,...)| | |No| | | |Downloading large files or databases|Final results or report (plots / tables)|Single (linear)| |Other|1-6 hours| | | | |No, is not available| |1-5|No|Collaborators|Yes|Yes|Email|Not open-sourced/open-licensed| |No|Fasta| | |No| |
|MS-B1H|Other|B1H screening|Identification of transcription factor binding motifs from B1H sequencing data|Francesco Tabaro|Boulard|EMBL Rome|2|Snakemake|Production|I do not version my workflow|Other|Manual testing (occasionally)| | | | | |Virtual environments (Conda, Venv, Renv,...)| | |No| | | |None, the workflow handles all the above|Final results or report (plots / tables)|Single (linear)| |Other|1-6 hours| | | | |No, is not available| |1-5|No|Myself|No|No|I don't|Not open-sourced/open-licensed| |No|Fastq| | |No| |
|Atlas baseline meta-analysis|Transcriptomics| |Workflow to merge meta data|Anil Thanki|Papatheodorou (Gene Expression Team)|EMBL-EBI|3|Snakemake|Experimental|Semantic Versioning|Monthly|Jenkins| | | | | |Virtual environments (Conda, Venv, Renv,...)| | |No| | | |None, the workflow handles all the above|Modified files to be used in other analysis|2 or more (e.g. alternative tools for key steps)| |Other|1-6 hours| | | | |No, is not available| |1-5|No|My group|Yes|No|Bug tracker|Other| |No|list of experiment ids, SDRF| | |Yes| |
|Literature-OpenTargets Pipeline|Other|Europe PMC scientific Literature article data|Lit-OTAR extracts Gene-Disease, Disease-Drug and Gene-Drug mentions from millions of Europe PMC abstracts and OA full-text research articles.|Santosh Tirunagari|Literature Services|EMBL-EBI|1|Other|Production|Release date|Daily|Manual testing (occasionally)| | | | | |Virtual environments (Conda, Venv, Renv,...)| | |No| | | |Downloading large files or databases|Final results or report (plots / tables)|2 or more (e.g. alternative tools for key steps)| |Daily|1-6 hours| | | | |Is public/privately available but not through a registry| |1-5|Yes|Collaborators|Yes|Yes|Email|GNU GPL| |Yes|Europe PMC Annotations API| | |Yes| |
|Literature-Metagenomics Pipeline|Metagenomics and other meta-omics| |metagenomics annotations pipeline was developed and integrated into Europe PMC to regularly enrich up-to-date ENA and MGnify metagenomics studies with metadata extracted from research articles. |Santosh Tirunagari|Literature Services|EMBL-EBI|1|Other|Production|Release date|Yearly|Manual testing (occasionally)| | | | | |Virtual environments (Conda, Venv, Renv,...)| | |No| | | |Downloading large files or databases|Modified files to be used in other analysis|2 or more (e.g. alternative tools for key steps)| |Yearly|6-24 hours| | | | |Is public/privately available but not through a registry| |1-5|Yes|Collaborators|Yes|Yes|Email|GNU GPL| |Yes|Europe PMC Annotations API| | |Yes| |
|Genome loader|Genomics and comparative genomics| |INSDC assembly and annotation loader into an Ensembl core db|Jorge Alvarez-Jarreta|Ensembl|EMBL-EBI|5|Nextflow|Production|I do not version my workflow| |Gitlab CI/CD| | | | | |Virtual environments (Conda, Venv, Renv,...)| | |No| | | |Parameterization / Settings| | | |Monthly|6-24 hours| | | | |Is public/privately available but not through a registry| | | | |Yes| | |Apache| |No| | | | | |
|Genome dumper|Genomics and comparative genomics| |Dumps the content of an Ensembl core db into flat files|Jorge Alvarez-Jarreta|Ensembl|EMBL-EBI|5|Nextflow|Production|I do not version my workflow| |Gitlab CI/CD| | | | | |Virtual environments (Conda, Venv, Renv,...)| | |No| | | |Parameterization / Settings| | | |Monthly|6-24 hours| | | | |Is public/privately available but not through a registry| | | | |Yes| | |Apache| |No| | | | | |
|Genome compare|Genomics and comparative genomics| |Compares the content of a Ensembl core db with another assembly/annotation available|Jorge Alvarez-Jarreta|Ensembl|EMBL-EBI|5|Nextflow|Production|I do not version my workflow| |Gitlab CI/CD| | | | | |Virtual environments (Conda, Venv, Renv,...)| | |No| | | |Parameterization / Settings| | | |Monthly|24-72 hours| | | | |Is public/privately available but not through a registry| | | | |Yes| | |Apache| |No| | | | | |
|bioimage analysis support|Bioimaging| |Automation of diverse bioimage analysis tasks|Christian Tischer|CBA|EMBL Heidelberg|2|Nextflow|Experimental|I do not version my workflow| |Manual testing (occasionally)| | | | | |Bundled static binaries| | |Graphical Processing Unit (GPU)| | | |Parameterization / Settings|Modified files to be used in other analysis| | |Weekly|<1 hour| | | | |Is public/privately available but not through a registry| |6-20|No|Collaborators|Yes|No|In person|BSD/MIT| |Yes| | | |No| |
|EBI Search indexer|Other|Metadata indexer pipeline|Gathering, indexing and publishing new data to EBI Search|Matt Pearce|Molecular Networks - EBI Search|EMBL-EBI|4|Other|Production|Semantic Versioning|Daily|Gitlab CI/CD| | | | | |Bundled static binaries| | |No| | | |None, the workflow handles all the above|Modified files to be used in other analysis|Single (linear)| |Daily|1-6 hours| | | | |No, is not available| |1-5|No|My group|No|Yes|Bug tracker|Apache| |No|XML, JSON| | |No| |
|Job Dispatcher Sequence Libraries Indexing (aka JD Data Update)|Other|Sequence library indexing (e.g. BLAST, FASTA and EMBOSS indexing)|A collection of Nextflow workflows are executed daily to index and publish new sequence library indexes (e.g. BLAST, FASTA and EMBOSS indexes) consumed by the Job Dispatcher Bioinformatics Tools|Fabio Madeira|SDO/TSC|EMBL-EBI|2|Nextflow|Production|I do not version my workflow|Daily|Gitlab CI/CD| | | | | |Container (Docker, Singularity)| | |No| | | |Manually installing dependencies|Modified files to be used in other analysis|Single (linear)| |Daily|6-24 hours| | | | |No, is not available| |6-20|No|My group|Yes|Yes|Forum / Discussion group|Apache| |Yes|FASTA, DAT, EMBL, SWISSPROT, JSON| | |No| |
|PDCM ETL| |Cancer models metadata and molecular data|ETL that generates the data for the catalog cancermodels.org|Mauricio Martinez Jimenez|Phenomics|EMBL-EBI|1|Other|Production|Semantic Versioning|Monthly|Gitlab CI/CD| | | | | |Container (Docker, Singularity)| | |No| | | |Parameterization / Settings|Final results or report (plots / tables)|Single (linear)| |Monthly|<1 hour| | | | |Is public/privately available but not through a registry| |1-5|No|My group|Yes|Yes|Bug tracker|Apache| |No| | | | | |
|EMICSS|Structural biology| |This service provides weekly updated cross-reference information for all EMDB entries, including both entry-level annotations and sample-level annotations.| Amudha Kumari Duraisamy|CS3DB|EMBL-EBI|2|Other|Production|Release date|Weekly|Jenkins| | | | | |Virtual environments (Conda, Venv, Renv,...)| | |No| | | |Downloading large files or databases|Final results or report (plots / tables)|Single (linear)| |Weekly|6-24 hours| | | | |No, is not available| |0|No|My group|Yes|Yes|I don't|Apache| |No| | | | | |
|Ensembl GenomeBrowser |Other|Data production |Data production for Ensembl Genome Browser|Marc Chakiachvili|Ensembl|EMBL-EBI|5|Nextflow|Production|I do not version my workflow| |Manual testing (occasionally)| | | | | |Modules| | |No| | | |Parameterization / Settings| |2 or more (e.g. alternative tools for key steps)| |Other|>72 hours| | | | |No, is not available| | |No|My unit|No|No|Forum / Discussion group|Apache| |No| | | | | |
|Ensembl THOAS|Other|Data provisioning|Public service to explore new ensembl Genomes|Marc Chakiachvili
Bethany Flint|Ensembl|EMBL-EBI|5|Nextflow|Production|I do not version my workflow| |Manual testing (occasionally)| | | | | |Modules| | |No| | | |Parameterization / Settings| | | | | | | | | | | | | | | | | | | | | | | | | |
|Ensembl Xref Reference Update|Genomics and comparative genomics| |Fetches files from sources external to Ensembl and prepare data to create Xrefs|Stefano Giorgetti|Ensembl|EMBL-EBI|1|Nextflow|Experimental|I do not version my workflow| |Travis| | | | | |Virtual environments (Conda, Venv, Renv,...)| | |No| | | |Parameterization / Settings|Modified files to be used in other analysis| | |Other|>72 hours| | | | |No, is not available| |1-5|Yes|My unit|No| |Forum / Discussion group|Apache| |No| | | | | |
|Ensembl Xref Mapping|Genomics and comparative genomics| |Uses data from "Xref references update" for refreshing/building mapping Ensembl data to external DBs|Stefano Giorgetti|Ensembl|EMBL-EBI|1|Nextflow|Experimental|I do not version my workflow| |Travis| | | | | |Virtual environments (Conda, Venv, Renv,...)| | |No| | | |Parameterization / Settings|Final results or report (plots / tables)| | |Other|>72 hours| | | | |No, is not available| |1-5|Yes|My unit|No| |Forum / Discussion group|Apache| |No| | | | | |
|Ensembl AlphaFold Mapping|Genomics and comparative genomics| |Maps Alphafold models to Ensembl translation objects|Stefano Giorgetti|Ensembl|EMBL-EBI|2|Nextflow|Experimental|I do not version my workflow| |Travis| | | | | |Virtual environments (Conda, Venv, Renv,...)| | |No| | | |Parameterization / Settings|Final results or report (plots / tables)| | |Other|1-6 hours| | | | |No, is not available| |1-5|Yes|My unit|No| |Forum / Discussion group|Apache| |No| | | | | |
|SORT-seq preprocessing|Transcriptomics| |Alignment filtering and count estimation for SORT-seq data|Maksim Kholmatov|Zaugg+SCB|EMBL Heidelberg|1|Snakemake|Experimental|I do not version my workflow|I'm not releasing updates|Manual testing (occasionally)| | | | | |Modules| | |No| | | |Parameterization / Settings|Final results or report (plots / tables)| | |Yearly|<1 hour| | | | |No, is not available| |0|No|Myself|No|No|In person|Not open-sourced/open-licensed| |No|FASTQ,BAM,RDS| | |No| |
|Ensembl BUSCO completeness assessment|Other|Genome/ proteome assessment|Fetches data from an ensembl sql database, runs BUSCO in genome or protein mode, write results to dir structure ready for rapid.ensembl.org ftp|Leanne Haggerty|Ensembl|EMBL-EBI|1|Nextflow|Production|I do not version my workflow|I'm not releasing updates|Manual testing (occasionally)| | | | | |Container (Docker, Singularity)| | |No| | | |Manually installing dependencies|Final results or report (plots / tables)|2 or more (e.g. alternative tools for key steps)| |Weekly|<1 hour| | | | |Is public/privately available but not through a registry| |1-5|Yes|My group|Yes|No|In person|Apache| |No| | | |No| |
|metaphlow|Metagenomics and other meta-omics| |Preprocessing and execution of taxonomic profiling via metaphlan3/4 on WGS reads|Christian Schudoma|Bork+SCB|EMBL Heidelberg|1|Nextflow|Production|Semantic Versioning|Other|Manual testing (occasionally)| | | | | |Container (Docker, Singularity)| | |No| | | |Parameterization / Settings|Final results or report (plots / tables)|Single (linear)| |Other|6-24 hours| | | | |Is public/privately available but not through a registry| | |No|My group|No|No|In person|BSD/MIT| |No|FASTQ| | |No| |
|nevermore|Metagenomics and other meta-omics| |Preprocessing and execution of functional profiling via gffquant on WGS reads|Christian Schudoma|Bork+SCB|EMBL Heidelberg|1|Nextflow|Production|Semantic Versioning|Other|Manual testing (occasionally)| | | | | |Container (Docker, Singularity)| | |No| | | |Parameterization / Settings|Final results or report (plots / tables)|Single (linear)| |Other|6-24 hours| | | | |Is public/privately available but not through a registry| |1-5|Yes|My group|No|No|In person|BSD/MIT| |No|FASTQ| | |No| |
|gq_flow|Metagenomics and other meta-omics| |Execution of functional profiling via gffquant on BAM files|Christian Schudoma|Bork+SCB|EMBL Heidelberg|1|Nextflow|Production|Semantic Versioning|Other|Manual testing (occasionally)| | | | | |Container (Docker, Singularity)| | |No| | | |Parameterization / Settings|Final results or report (plots / tables)|Single (linear)| |Other|6-24 hours| | | | |Is public/privately available but not through a registry| |1-5|Yes|My group|No|No|In person|BSD/MIT| |No|BAM,SAM| | |No| |
|MAGs annotation pipeline|Metagenomics and other meta-omics| |A pipeline to perform taxonomic and functional annotation and to generate a catalogue from a set of isolate and/or metagenome-assembled genomes (MAGs)|Martin Beracochea|Microbiome Informatics|EMBL-EBI|4|Nextflow|Production|Semantic Versioning|Other|Jenkins| | | | | |Container (Docker, Singularity)| | |Large memory (256GB+)| | | |Downloading large files or databases|Final results or report (plots / tables)|2 or more (e.g. alternative tools for key steps)| |Monthly|>72 hours| | | | |WorkflowHub| | |No|My group|Yes|Yes|Bug tracker|Apache| |Yes|FASTA| | | | |
|MGnify analysiss pipeline|Metagenomics and other meta-omics| |MGnify pipeline version 5.0|Martin Beracochea|Microbiome Informatics|EMBL-EBI|5|CWL|Production|Semantic Versioning|Yearly|Jenkins| | | | | |Other| | |No| | | |Downloading large files or databases|Final results or report (plots / tables)|2 or more (e.g. alternative tools for key steps)| |Daily|24-72 hours| | | | |WorkflowHub| | |No|My group|Yes|Yes|Email|Apache| |Yes|FASTQ, FASTA| | | | |
|MGnify raw reads taxonomic profiling pipeline|Metagenomics and other meta-omics| |MGnify raw reads taxonomic profiling pipeline|Martin Beracochea|Microbiome Informatics|EMBL-EBI|2|Nextflow|Production|Semantic Versioning|Other|Jenkins| | | | | |Container (Docker, Singularity)| | |No| | | |Downloading large files or databases|Final results or report (plots / tables)|Single (linear)| |Weekly|6-24 hours| | | | |WorkflowHub| | |No|My group|Yes|Yes|Bug tracker|Apache| |Yes|FASTQ| | | | |
|MoMofy|Metagenomics and other meta-omics| |MoMofy: Module for integrative Mobilome prediction|Martin Beracochea|Microbiome Informatics|EMBL-EBI|2|Nextflow|Experimental|Semantic Versioning|Other|Jenkins| | | | | |Container (Docker, Singularity)| | |No| | | |Downloading large files or databases|Final results or report (plots / tables)|Single (linear)| |Other|6-24 hours| | | | |WorkflowHub| | |No|My group|Yes|Yes|Bug tracker|Apache| |Yes|FASTA| | | | |
|VIRify|Metagenomics and other meta-omics| |VIRify is a recently developed pipeline for the detection, annotation, and taxonomic classification of viral contigs in metagenomic and metatranscriptomic assemblies.|Martin Beracochea|Microbiome Informatics|EMBL-EBI|4|Nextflow|Production|Semantic Versioning|Other|Jenkins| | | | | |Container (Docker, Singularity)| | |No| | | |Parameterization / Settings|Final results or report (plots / tables)|2 or more (e.g. alternative tools for key steps)| |Monthly|1-6 hours| | | | |WorkflowHub| |1-5|Yes|My group|Yes|Yes|Bug tracker|Apache| |Yes|FASTA, FASTQ| | | | |
|SMAP|Bioimaging| |Workflow to analyze super-resolution images acquired by SMLM. Includes single-molecule fitting, rendering, post-processing and further analysis|Jonas Ries|Ries, CBB|EMBL Heidelberg|2|Other|Experimental|I do not version my workflow|Monthly|Manual testing (occasionally)| | | | | |Virtual environments (Conda, Venv, Renv,...)| | |Graphical Processing Unit (GPU)| | | |Parameterization / Settings|Final results or report (plots / tables)|2 or more (e.g. alternative tools for key steps)| |Daily|<1 hour| | | | |Is public/privately available but not through a registry| |20-50|Yes|Myself|Yes|No|Email|GNU GPL| |Yes|input Tiff, output various: tiff, h5, matlab binaries| | |No| |
|BENGAL|Transcriptomics| |BENGAL: BENchmarking strateGies for cross-species integrAtion of singLe-cell RNA sequencing data. Runs scRNA-seq integration of data from multiple species and examine the output.|Yuyao Song|Papatheodorou (research)|EMBL-EBI|1|Nextflow|Experimental|Semantic Versioning|Other|Manual testing (occasionally)| | | | | |Virtual environments (Conda, Venv, Renv,...)| | |Graphical Processing Unit (GPU)| | | |Parameterization / Settings|Final results or report (plots / tables)|Single (linear)| |Monthly|1-6 hours| | | | |Is public/privately available but not through a registry| | | |My group|Yes|No|Bug tracker|BSD/MIT| |Yes|input anndata, output anndata, csv| | | | |
|CATD(Critical Assesment of Trascriptomic Deconvolution)|Transcriptomics| |The CATD pipeline is a benchmarking pipeline meant to facilitate the assessment of cell-type deconvolution methods(29 now) across different simulation scenarios in a standardised way. |Anna Vathrakokoili Pournara|Papatheodorou Research Group|EMBL-EBI|1|Snakemake|Experimental|Release date|Other|Manual testing (occasionally)| | | | | |Virtual environments (Conda, Venv, Renv,...)| | |Graphical Processing Unit (GPU)| | | |Parameterization / Settings|Final results or report (plots / tables)|2 or more (e.g. alternative tools for key steps)| |Monthly|6-24 hours| | | | |Is public/privately available but not through a registry| |1-5|Yes|My group|Yes|Yes|Bug tracker|BSD/MIT| |Yes|inputs : anndata / Seurat Object, outputs: .rds , .csv for tables and pdf for plots| | |No| |
|Protein Sequence Embeddings|Other|Protein sequence embeddings|Workflow to generate protein sequence embeddings (using T5 model) for uniprotkb sequences and publish to users on FTP and/or store into db for API access. |Abdulrahman Hussein|UniProt, AA|EMBL-EBI|1|Nextflow|Production|I do not version my workflow|Other|Manual testing (occasionally)| | | | | |Virtual environments (Conda, Venv, Renv,...)| | |Graphical Processing Unit (GPU)| | | |Parameterization / Settings|Final results or report (plots / tables)|2 or more (e.g. alternative tools for key steps)| |Other|6-24 hours| | | | |No, is not available| | |No|My unit|Yes|No|I don't| | |Yes|input: fasta, output: h5| | |Yes| |
|Quarto Book Publishing|Other|Publishing documentation|Workflow that compiles a complete HTML book from Quarto source files|Mike Smith|Data Science Centre|EMBL Heidelberg|1|Other|Production|I do not version my workflow|Other|GitHub actions| | | | | |Container (Docker, Singularity)| | |No| | | |None, the workflow handles all the above|Final results or report (plots / tables)|Single (linear)| |Weekly|<1 hour| | | | |Is public/privately available but not through a registry| |0|No|My group|Yes|No|In person|Not open-sourced/open-licensed| |No|Qmd| | |No| |
|Adaptive Feedback Microscopy workflow development and support|Bioimaging| |A toolbox containing several preconfigured workflows and the capability to design custom workflows for users, posibility to run in conjunction with several commercial microscope systems|Aliaksandr Halavatyi|ALMF|EMBL Heidelberg|2|Other|Production|Other|Monthly|Manual testing (occasionally)| | | | | |Other| | |Other| | | |Parameterization / Settings|Modified files to be used in other analysis|2 or more (e.g. alternative tools for key steps)| |Weekly|<1 hour| | | | |Is public/privately available but not through a registry| |6-20|Yes|Collaborators|Yes|No|In person|Not open-sourced/open-licensed| |Yes|input: images and workflow parameters; output:images, experiment information files and log files| | |No| |
|CRIMS-Lab|Structural biology| |A web interface to monitor and pilot your crystallization experiments|Jose Antonio Marquez|HTXLab|EMBL Grenoble|3|Other|Production|Other|Monthly|Manual testing (occasionally)| | | | | |Container (Docker, Singularity)| | |Other| | | |Parameterization / Settings|Final results or report (plots / tables)|2 or more (e.g. alternative tools for key steps)| |Daily|>72 hours| | | | |No, is not available| |20-50|Yes|My group|Yes|No|Email|BSD/MIT| |Yes|No input. Output: crystals images, csv files, tables| | |No| |
|CRIMS-Connect|Structural biology| |A web interface to monitor and pilot your shipments to synchrotrons|Jose Antonio Marquez|HTXLab|EMBL Grenoble|3|Other|Production|Other|Monthly|Manual testing (occasionally)| | | | | |Container (Docker, Singularity)| | |Other| | | |Parameterization / Settings|Final results or report (plots / tables)|Single (linear)| |Weekly|6-24 hours| | | | |No, is not available| |1-5|Yes|My group|Yes|No|Email|BSD/MIT| |Yes|Input: Either data from Crims-Lab, or csv file. Output: diffraction images, data collection metadata.| | |No| |
|CRIMS-Pipedream|Structural biology| |A web interface to monitor and pilot data processing workflow|Jose Antonio Marquez|HTXLab|EMBL Grenoble|3|Other|Production|Other|Monthly|Manual testing (occasionally)| | | | | |Container (Docker, Singularity)| | |Other| | | |Parameterization / Settings|Final results or report (plots / tables)|Single (linear)| |Weekly|1-6 hours| | | | |No, is not available| |1-5|Yes|My group|Yes|No|Email|BSD/MIT| |Yes|Input: Either data from CRIMS-Connect, or csv file, diffraction images, pdb, mtz, cif files.| | |No| |
